torch/onnx/__init__.py

```diff
@@ -303,8 +303,10 @@ def forward(self, x):
             If a custom opset is referenced by ``model`` but not mentioned in this dictionary,
             the opset version is set to 1.
 
-        enable_onnx_checker (bool, default True): If True the onnx model checker will be run
-            to ensure the exported model is a valid ONNX model.
+        enable_onnx_checker (bool, default True): [Deprecated and ignored. Will be removed in next
```

Comment:
Write just:
`enable_onnx_checker (bool, default True): Deprecated and ignored. Will be removed in next PyTorch release.`

All the other text here is misleading and unnecessary.

Reply:
Since this parameter will still be there in current PyTorch release, don't we keep those explanations for its usage?

In addition, we've made a change that even the ONNX graph is invalid, the proto file will still be generated. We should also let user to be aware of this expected behavior. Is it right?

Reply:
> don't we keep those explanations for its usage

No, any explanation is misleading because the argument is totally ignored.

> even the ONNX graph is invalid, the proto file will still be generated

Please document this in the "Raises:" section of the doc string.